lib/src/form_builder_validators.dart

```diff
@@ -220,11 +222,12 @@ class FormBuilderValidators {
   /// [FormFieldValidator] that requires the field's value to be a valid double.
   static FormFieldValidator double_(
     BuildContext context, {
-    String errorText = 'Value must be a double.',
+    String errorText,
   }) {
     return (valueCandidate) {
       if (double.tryParse(valueCandidate ?? '') == null) {
```

Comment:
This:
```dart
if (double.tryParse(valueCandidate ?? '') == null)
```
will erroneously trigger an empty string.

Reply:
Done.